modules/statistics/RTPStatsCollector.js

```diff
@@ -749,9 +749,12 @@ StatsCollector.prototype.processAudioLevelReport = function () {
     for (var idx in this.currentAudioLevelsReport) {
         var now = this.currentAudioLevelsReport[idx];
 
-        //if we don't have "packetsReceived" this is local stream
-        if (now.type != 'ssrc' || !getStatValue(now, 'packetsReceived')) {
+        if (now.type != 'ssrc')
             continue;
+
+        var isLocal = false;
+        if (!getStatValue(now, 'packetsReceived')) {
+            isLocal = true;
```

Comment:
That's a very long way of saying `var isLocal = !getStatValue(now, 'packetsReceived');`.